workflow.md

```diff
@@ -54,6 +54,12 @@ project manager would do it himself). Do not merge your own PRs!
 Please describe in PR what have you done, if it is tested or not, any additional
 information.
 
+## Merging Pull requests
+
+After review by another team member, the teammate merging the PR should merge
+`master` into the Pull request's branch *without* the `--no-ff` option, and then
```

Comment:
@kantp we discussed it in private and decided that it's better to reformulate (and slightly change meaning) as:
> Generally pull request's branch should be merged into _master_ , but if you are sure that rebasing this branch onto _master_ is better (for instance this is a trivial change which doesn't deserve branching) then you can rebase instead.

However, let's wait until someone merges this pull request, maybe they will have something to say.